pkg/kubelet/kubelet.go

```diff
@@ -1853,28 +1831,13 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 			klog.V(2).Infof("SyncLoop (DELETE, %q): %q", u.Source, format.Pods(u.Pods))
 			// DELETE is treated as a UPDATE because of graceful deletion.
 			handler.HandlePodUpdates(u.Pods)
-		case kubetypes.RESTORE:
-			klog.V(2).Infof("SyncLoop (RESTORE, %q): %q", u.Source, format.Pods(u.Pods))
-			// These are pods restored from the checkpoint. Treat them as new
-			// pods.
-			handler.HandlePodAdditions(u.Pods)
 		case kubetypes.SET:
 			// TODO: Do we want to support this?
 			klog.Errorf("Kubelet does not support snapshot update")
 		}
```

Comment:
should there be a default case that at least logs receiving unhandled op types?

Reply:
Is it worth to test this code branch, mocking klog maybe?
```
func TestInvalidEventSyncLoop(t *testing.T) {
       testKubelet := newTestKubelet(t, false /* controllerAttachDetachEnabled */)
       defer testKubelet.Cleanup()

       kubelet := testKubelet.kubelet
      kubelet.runtimeState.setRuntimeSync(time.Now())
       kubelet.resyncInterval = time.Second * 30

       ch := make(chan kubetypes.PodUpdate)
       // add an invalid operation
       go func() { ch <- kubetypes.PodUpdate{Op: 1000, Source: kubetypes.ApiserverSource} }()
       defer close(ch)

       kubelet.syncLoopIteration(ch, kubelet, make(chan time.Time), make(chan time.Time), make(chan *pleg.PodLifecycleEvent, 1))
       // TODO - mocked klog call assertion
}
```

Reply:
@liggitt any thoughts on this?

Reply:
mocking klog seems like overkill here... I mostly just care about not silently dropping unexpected cases